README.md

```diff
@@ -26,28 +26,27 @@ Questions on how to add your app to Groot or use the Groot API:
     sudo apt install repo
 
     ```
-2. Make a directory to house your groot work
+2. Clone this repository
```

Comment:
You should not need to clone the repo, the repo sync should pull down deploy as well now. Just make a directory, init and sync